### PR TITLE
py-cwl-registry: Release 0.4.6

### PR DESCRIPTION
**Pull request by eleftherioszisis**

### CODE REVIEW

File: bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class PyCwlRegistry(PythonPackage):
     depends_on("py-nexusforge@0.8.1:", type=("build", "run"))
     depends_on("py-bba-data-push@1.0.4:", type=("build", "run"))
     depends_on("py-cwl-luigi@0.3.1:", type=("build", "run"))
-    depends_on("py-entity-management@1.2.26:", type=("build", "run"))
+    depends_on("py-entity-management@1.2.27:", type=("build", "run"))
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]:
```
cwl-registry 0.4.5 has requirement entity-management==1.2.27.dev0, but you have entity-management 1.2.27.
```
:shrug:

Comment (eleftherioszisis) [reply bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: Yeah, my bad..

File: bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py [hunk 2]
```diff
@@ -10,12 +10,12 @@ class PyCwlRegistry(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/nse/cwl-registry.git"
 
     version("develop", branch="main")
-    version("0.4.4", tag="cwl-registry-v0.4.4")
+    version("0.4.6.dev2", tag="cwl-registry-v0.4.6.dev2")
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: This lead to bad results the last time around.

Comment (eleftherioszisis) [reply bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: Thanks for bringing this up. I removed the dev2 to test the 0.4.6.dev2 as "0.4.6".

File: bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py [hunk 3]
```diff
@@ -10,12 +10,12 @@ class PyCwlRegistry(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/nse/cwl-registry.git"
 
     version("develop", branch="main")
-    version("0.4.4", tag="cwl-registry-v0.4.4")
+    version("0.4.6", tag="cwl-registry-v0.4.6.dev4")
```
Comment (genric) [inline bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: why `dev4`?

Comment (eleftherioszisis) [reply bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: Because I am testing it :)

Comment (genric) [reply bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py]: Then please mark this as `DRAFT` ;)

**Comment by matz-e** [comment]
Probably needs a rebase

**Review by genric — APPROVED** [review]

**Review by matz-e — APPROVED** [review]

**Pull request merged**